docs/core/config/auth/caching.md

```diff
@@ -57,11 +57,11 @@ specifying the cache key (via `cache_key`):
 - [[link,auth_pam]]
 - [[link,auth_bsd]]
```

Comment:
not true. the only one that supports specifying cache_key is passdb/userdb lua.

Reply:
Again: This was not touched in the config-rewrite branch that I am porting. This may be another JIRA ticket. Even if this is a simple fix, I don't feel comfortable inflating these PRs much more than already happened.

Reply:
DOV-7009